addons/auth_signup/models/res_users.py

```diff
@@ -221,8 +221,10 @@ def send_unregistered_user_reminder(self, after_days=5):
 
         # For sending mail to all the invitors about their invited users
         for user in invited_users:
-            template = self.env.ref('auth_signup.mail_template_data_unregistered_users').with_context(dbname=self._cr.dbname, invited_users=invited_users[user])
-            template.send_mail(user, email_layout_xmlid='mail.mail_notification_light', force_send=False)
+            email_template = self.env.ref('auth_signup.mail_template_data_unregistered_users', raise_if_not_found=False)
+            if email_template:
```

Comment:
I would move getting the `email_template` and the `if` in the beginning of the function before any work is done. Sending emails is the whole point of this cron, shouldn't we log something if the cron can't do its work ?